src/twisted/python/_url.py

```diff
@@ -528,9 +528,10 @@ def child(self, *segments):
 
         For example::
 
-            >>> (URL.fromText(u"http://localhost/a/b?x=y")
-                 .child(u"c", u"d").asText())
-            u'http://localhost/a/b/c?x=y'
+            >>> u = URL.fromText(u"http://localhost/a/b?x=y")
+            >>> v = u.child(u"c", u"d")
+            >>> print(v.asText())
+            u'http://localhost/a/b/c/d?x=y'
```

Comment:
Uh. 2 dumb questions.

1) epydoc doesn't have doctests then I take it?
2) Is the policy that we use Py2 reprs then? I assume the answer must be yes I suppose.

Regardless, lgtm.